Firestore/core/src/model/object_value.cc

```diff
@@ -72,6 +73,97 @@ static google_firestore_v1_MapValue_FieldsEntry* FindEntry(
   return found.first;
 }
 
+size_t CalculateSizeOfUnion(
+    google_firestore_v1_MapValue* map_value,
```

Comment:
Nit: you can pass the map by const reference.

Reply:
Fixed